lib/api/_loaders/static.js

```diff
@@ -215,7 +220,7 @@ module.exports = class StaticAssert {
       };
 
       this.nightwatchInstance.setApiMethod(prop, namespace || 'assert', (function(prop) {
-        return this.createStaticAssertion(prop, true);
+        return this.createStaticAssertion(prop, true, apiToReturn);
```

Comment:
I think this is correct, assert is anyways returning nightwatchAPI [here](https://github.com/nightwatchjs/nightwatch/pull/3779/files#diff-2ee35a45c421f9afae7f862ec1e270e0cce2a0ac46fe3aad8c69d84ae1c899dbL193). This line overrides that to return pageObject instead.

Reply:
Assert should return anything. I guess. Asserts are not meant to be chained with other commands.  @swrdfish the linked code I think points to the same PR which was part of this feature I think. 
@beatfactor thoughts?

Reply:
> the code change linked points to the PR which went in as part of the same page-object feature.

Nope, it was there earlier. The only change made there in the last PR was `return this.api` -> `return apiToReturn || this.api`, and `return this.api` has been there for the last 4 years (added [here](https://github.com/nightwatchjs/nightwatch/commit/0d3ceb53fb67f3039e14d7c8bf675ccf82a0ff5a#diff-2ee35a45c421f9afae7f862ec1e270e0cce2a0ac46fe3aad8c69d84ae1c899dbR122)).

And I think the `assert` API is meant to be chained with the other commands, as it is being done in most of our popular examples (like ecosia and duckDuckGo).

And related to `assert` not returning anything, I personally find it useful to have assert return whatever it is testing on. For example, if `assert.titleContains('some title')` is failing, I can just console log the result retuned by `assert` to get the updated title of the page, instead of using entirely separate `.getTitle()` command to get the title and update it in the test.